Errno error on windows/cygwin

I noticed this error while parsing the CI log for a windows runner
```
2020-08-03T02:56:03.9860020Z gmake[1]: Entering directory 'd:/a/perl/perl/ext/Errno'
2020-08-03T02:56:04.4766572Z "d:\a\perl\perl\miniperl.exe" "-I..\..\lib" Errno_pm.PL Errno.pm 
2020-08-03T02:56:05.3842816Z In file included from includes.c:1:
2020-08-03T02:56:05.3872758Z C:/strawberry/c/lib/gcc/x86_64-w64-mingw32/8.3.0/include/avx512dqintrin.h:25:2: error: #error "Never use <avx512dqintrin.h> directly; include <immintrin.h> instead."
2020-08-03T02:56:05.5762837Z  #error "Never use <avx512dqintrin.h> directly; include <immintrin.h> instead."
2020-08-03T02:56:05.5837917Z   ^~~~~
2020-08-03T02:56:05.6561346Z In file included from includes.c:10:
2020-08-03T02:56:05.7733058Z C:/strawberry/c/lib/gcc/x86_64-w64-mingw32/8.3.0/include/xtestintrin.h:25:3: error: #error "Never use <xtestintrin.h> directly; include <immintrin.h> instead."
2020-08-03T02:56:05.8453054Z  # error "Never use <xtestintrin.h> directly; include <immintrin.h> instead."
2020-08-03T02:56:05.8732496Z    ^~~~~
2020-08-03T02:56:06.6634861Z gmake[1]: Leaving directory 'd:/a/perl/perl/ext/Errno'
```
full log available at https://pipelines.actions.githubusercontent.com/StwKAELrIIfJZjsjvFeJdim2u3AOp8ez6qvldOETh5vCDDWoz5/_apis/pipelines/1/runs/305/signedlogcontent/24?urlExpires=2020-08-03T04%3A56%3A23.5820001Z&urlSigningMethod=HMACV1&urlSignature=gbS4rY1tn8pCqAYy5pW3xLRld2r6pQOEIaKaAgW4PwQ%3D
I think we can also find the error/warnings from the most recent runs

suggested patch
```diff
diff --git a/ext/Errno/Errno_pm.PL b/ext/Errno/Errno_pm.PL
index 4f8acfee23..c5ba164acb 100644
--- a/ext/Errno/Errno_pm.PL
+++ b/ext/Errno/Errno_pm.PL
@@ -18,11 +18,19 @@ if ($Config{gccversion} ne '' && $^O eq 'MSWin32') {
     # MinGW complains "warning: #pragma system_header ignored outside include
     # file" if the header files are processed individually, so include them
     # all in .c file and process that instead.
+    my %seen;
     open INCS, '>', 'includes.c' or
        die "Cannot open includes.c";
     foreach $file (@files) {
        next if $file eq 'errno.c';
        next unless -f $file;
+       if ( $file eq 'avx512vpopcntdqvlintrin.h' || $file eq 'avx512bwintrin.h' ) {
+               # "Never use <avx512bwintrin.h> directly; include <immintrin.h> instead."
+               # "Never use <avx512vpopcntdqvlintrin.h> directly; include <immintrin.h> instead."
+               # "Never use <avx512vpopcntdqvlintrin.h> directly; include <immintrin.h> instead."
+               $file = 'immintrin.h';
+       }
+       next if ++$seen{$file} > 1;
        print INCS qq[#include "$file"\n];
     }
     close INCS;
```

--- a/ext/Errno/Errno_pm.PL
+++ b/ext/Errno/Errno_pm.PL
@@ -2,7 +2,7 @@ use ExtUtils::MakeMaker;
 use Config;
 use strict;
 
-our $VERSION = "1.31";
+our $VERSION = "1.32";
 
 my %err = ();
 
@@ -18,11 +18,18 @@ if ($Config{gccversion} ne '' && $^O eq 'MSWin32') {
     # MinGW complains "warning: #pragma system_header ignored outside include
     # file" if the header files are processed individually, so include them
     # all in .c file and process that instead.
+    my %seen;
     open INCS, '>', 'includes.c' or
 	die "Cannot open includes.c";
     foreach $file (@files) {
 	next if $file eq 'errno.c';
 	next unless -f $file;
+	if ( $file eq 'avx512vpopcntdqvlintrin.h' || $file eq 'avx512bwintrin.h' ) {
+		# "Never use <avx512bwintrin.h> directly; include <immintrin.h> instead."
+		# "Never use <avx512vpopcntdqvlintrin.h> directly; include <immintrin.h> instead."
+		$file = 'immintrin.h';
+	}
+	next if ++$seen{$file} > 1;
 	print INCS qq[#include "$file"\n];
     }
     close INCS;

